### DISCUSSION TITLE
Codespaces—Git coding fast

### DISCUSSION THREAD

**Discussion by samus-aran** [community/events#34] [Announcements]
Codespaces helps you get coding faster and focus on building what matters together. In a single click, launch any GitHub repository with your favorite editor—backed with compute and terminal access—customized with your themes, preferences, and dotfiles. April Leonard will show you how to explore, contribute, and start coding with infinite, isolated, secure, and fast developer environments. [Check out this session Live](https://watch.githubuniverse.com/live/dayOneTrackOne)!

**Comment by burhan** [comment]
Heartwarming to see even the greatest struggle with git 😊, makes me feel less of a dunce when I goof up.

**Reply by hrthealer** [reply]
Yes I agree, it is a project that still has some things to work on... The Copilot movement is helping the speed of completing code, the latter is how much speed for debugging ?  The balance of these works should all come together and work in balance and deploy with as little errors as possible.  I am just @ the first few rungs of this new adventure, have a GR8 time.

**Comment by JesseStorms** [comment]
Seems extremely useful for students. I've had several assignments where I had issues which would be solved by codespaces. Are there plans to give students extra time/resources through the Education pack?

**Comment by CryptoHighway84** [comment]
This was a great crash course space. I enjoyed the event!

**Comment by Manamama** [comment]
I am based in the EU. i can see only:
Codespaces is rolling out!
You don't have access yet, but in the meantime, you can learn more about Codespaces.

A bug or a feature?
